core/src/main/resources/benchmark-spark-mr.xml

```diff
@@ -0,0 +1,100 @@
+<!-- RadarGun 2.2 benchmark -->
+<benchmark xmlns="urn:radargun:benchmark:2.2">
+
+   <!-- Specifies where should the master open socket -->
+   <master bindAddress="${master.address:127.0.0.1}" port="${master.port:2103}"/>
+
+   <!-- List of cluster configurations where the benchmark should run -->
+   <clusters>
+      <cluster size="5">
+         <group name="master" size="1" />
+         <group name="worker" size="1" />
+         <group name="driver" size="1" />
+         <group name="ispn-server" size="1" />
+         <group name="ispn-client" size="1" />
+      </cluster>
+   </clusters>
+
+   <!-- List of configurations of the services -->
+   <configurations>
+      <config name="Spark">
+         <!-- Configuration for the Spark master group -->
+         <setup group="master" plugin="spark">
+            <master xmlns="urn:radargun:plugins:spark:2.2" home="${env.SPARK_HOME}">
+            </master>
+         </setup>
+         <!-- Configuration for the Spark worker group -->
+         <setup group="worker" plugin="spark">
+            <worker xmlns="urn:radargun:plugins:spark:2.2" home="${env.SPARK_HOME}">
+               <host>localhost.localdomain</host>
+               <port>7077</port>
+            </worker>
+         </setup>
+         <!-- Configuration for the Spark driver group -->
+         <setup group="driver" plugin="spark">
+            <driver xmlns="urn:radargun:plugins:spark:2.2">
+               <host>localhost.localdomain</host>
+               <port>7077</port>
+               <source-class>org.radargun.service.demo.ispn.WordCountSource</source-class>
+               <source-params>setHotrodServer:${hotrodServer:127.0.0.1};setHotrodPort:${hotrodPort:11322}</source-params>
+               <properties>
+                  <key-value key="spark.serializer" value="org.apache.spark.serializer.KryoSerializer"/>
```

Comment:
If you you have outer tag &lt;properties&gt;, I would suggest rather inner element `<property name="..." value="...";/>`